src/serialMIDI.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "midi_Namespace.h"
+
+BEGIN_MIDI_NAMESPACE
+
+template <class SerialPort>
+class serialMIDI
```

Comment:
Can you name it with an Uppercase first letter ? Just to be consistent with the rest of the coding style, thanks ! :)